src/main/pages/workspace-admin/recipes.adoc

```diff
@@ -100,34 +143,28 @@ networks:
   driver: bridge
 ----
 
-Che does not yet currently support compose networks. We do, however, support the use of aliases through `links` command.
-
 *Hostname*
 
-Hostname is not supported and the machine’s name will be used for the hostname. User can use `links` aliases syntax to add additional hostname(s) to a machine.
+The hostname is not supported.  The machine’s name is used for the hostname. You can use `links` aliases syntax to add additional hostnames to a machine.
 
-*Ports*
+*Binding ports*
 
-Binding ports to the host system is not supported. This is done to ensure that each container does not used already assigned host ports because many developers could be running the same compose file on the same Che server host at the same time. Users can add link:servers.html[servers] to machines though.
+Binding ports to the host system is not supported.  You must ensure that each container does not use already assigned host ports.  Many developers can be running the same Compose file on the same Che server host at the same time. You can add link:servers.html[servers] to the machines.
```

Comment:
- This edit changes orig. meaning (leaving out 'because').
- don't use speculative lang. like "many developers can be" -- just state the reason why the binding is not supported, and how it can be worked around

Reply:
How about this:

Binding ports to the host system is not supported to ensure that each container does not use already assigned host ports.  You can add link:servers.html[servers] to machines instead of binding ports to the host system.